simple/stats/stat_var_hierarchy_generator.py

```diff
@@ -136,7 +152,7 @@ def __init__(self, svg_id: str, svg_name: str) -> None:
     # Maintaining order maintains results consistency and helps with tests.
     self.sv_ids: dict[str, bool] = {}
     self.parent_svg_ids: dict[str, bool] = {}
-    self.child_svg_ids: dict[str, bool] = {}
+    self.child_svg_id_2_specialized_name: dict[str, str] = {}
```

Comment:
How does this get surfaced?

Reply:
Currently it does not. I'll generate a svg cache in a follow-on PR and it will get surfaced there.